tox.ini

```diff
@@ -68,7 +68,7 @@ commands =
 description = Run all integration tests
 deps =
     pytest
-    juju
+    juju==2.9.11 # juju 3.0.0 has issues with retrieving action results
```

Comment:
JFMI, why do we lock such an old juju version? 2.9.37 is the latest as for today.

Reply:
This is the last 2.9 version of _libjuju_, which doesn't follow the same version as juju :)